src/app/containers/AssignmentDeleteModal.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import {
+    Button,
+    Glyphicon
+} from 'react-bootstrap';
+import ModalWrapper from './ModalWrapper';
+import { connect } from 'react-redux';
+import { deleteAssignmentTemplate } from '../modules/assignments/actions';
+
+export interface AssignmentDeleteModalProps {
+    templateId: number;
+    deleteAssignment?: (id: number) => void;
+}
+
+class AssignmentDeleteModal extends React.Component<AssignmentDeleteModalProps>{
+    render() {
+        const { deleteAssignment, templateId } = this.props;
+        return (
+            <div>
+                <ModalWrapper
```

Comment:
This could be abstracted into a Prompt component class in the same way we created the ModalWrapper.  Just a specialized version of the modal for prompting and then doing an action.  Not required, just to keep in mind the next time you have to do it and consider copying it.